apidoc/Titanium/UI/TextArea.yml

```diff
@@ -45,6 +45,28 @@ methods:
         See also: [add](Titanium.UI.TextArea.add), <Titanium.UI.View.add>.
     platforms: [iphone, ipad]
     
+  - name: padding
+    summary: Sets the left and right padding of this TextArea. The text will always be vertically centered.
+    type: Padding
+    platforms: [android, ios, ipad]
+    since: 5.4.0
+    
+  - name: paddingLeft
+    summary: Left padding of this text field.
+    type: Number
+    platforms: [iphone, ipad]
+    deprecated:
+        since: "5.4.0"
+        notes: Use padding instead    
+    
+  - name: paddingRight
```

Comment:
So `paddingLeft` and `paddingRight` haven't been documented for `Ti.UI.TextArea` before?